steps/codecov/3.0.0/step.yml

```diff
@@ -0,0 +1,43 @@
+title: Codecov
+summary: Upload your code coverage files to Codecov.io
+description: |-
+  Reduce technical debt with visualized test performance,
+  faster code reviews and workflow enhancements.
+  Now you can upload your code coverage files to Codecov every time you kick off a build!
+website: https://codecov.io
+source_code_url: https://github.com/codecov/codecov-bitrise
+support_url: https://community.codecov.io
+published_at: 2021-08-06T12:32:31.826056-04:00
+source:
+  git: https://github.com/codecov/codecov-bitrise.git
+  commit: 31f43b1ede55e770d642280d97e16e4a166c4d16
+host_os_tags:
+- osx-10.10
+- ubuntu-14.04
+type_tags:
+- test
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: true
+inputs:
+- CODECOV_TOKEN: null
+  opts:
+    description: |
+      Codecov.io repository upload token
+    is_required: true
+    is_sensitive: true
+    title: Codecov.io token
+- OS: null
```

Comment:
As Os is a predefined list, please use `value_options` to list possible values (https://devcenter.bitrise.io/contributors/create-your-own-step/#configuring-step-inputs).
This allows the values to be selected on the Workflow Editor UI. In addition to this it would be preferable to add validation that `OS` is a valid option inside the Step code. (As one still could pass an empty or invalid value by manually editing *bitrise.yml*).

Ideally, automatically detecting the OS (linux/macos) would be the most ideal as setting this input would not be required. This is not a blocker for now, but it would be great if you could include it in the next release.